src/awkward/_v2/_connect/cuda/cuda_kernels/awkward_BitMaskedArray_to_ByteMaskedArray.cu

```diff
@@ -0,0 +1,50 @@
+// BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+__global__ void
```

Comment:
Would `__device__` work as well? None of these kernels are going to run on the CPU, right? Declaring them as GPU-only should cut the compilation time in half, and now that matters because they're being compiled while the user waits.

Also, if there's some reason why one of these kernels can't be compiled on the CPU—because you use some intrinsic, for instance—then avoiding the attempt removes that difficulty.

Reply:
A `__device__` function can only be called from the GPU itself. We define `__global__` so that the kernel can be invoked from the CPU and the GPU (The first call to the kernel is always from the CPU). Regardless of the flag, these kernels cannot run on a CPU. These flags exist as invocation rules.